dbms/programs/client/Client.cpp

```diff
@@ -1537,6 +1550,13 @@ class Client : public Poco::Util::Application
         std::cout << DBMS_NAME << " client version " << VERSION_STRING << VERSION_OFFICIAL << "." << std::endl;
     }
 
+    static std::pair<String, String> parseParam(const String & s)
+    {
+        size_t pos = s.find('_') + 1;
+        /// Cut two first dash "--" and divide arg from name and value
+        return std::make_pair(s.substr(2, pos - 2), s.substr(pos));
```

Comment:
Overflow is possible.

Reply:
It's impossible, string begins with "--param"